### ISSUE TITLE
ERROR: Cannot uninstall 'llvmlite'

### ISSUE THREAD

**Issue by MaXDL4Phys** [rkansal47/MPGAN#8]
Hi, 

while trying to run `docker build .` I get the error `ERROR: Cannot uninstall 'llvmlite''.`
This happens during the main build and during a step-by-step installation after the command: `pip install jetnet`.
Any solution? Thanks in advance!

**Comment by rkansal47** [comment]
Hmm, I haven't encountered this issue before. What OS version and Python version are you using?

**Comment by MaXDL4Phys** [comment]
Hi Raghav,
Probably the problems were due to incompatibilities in my OS (Ubuntu 18
LTS) and version Python (3.8) of Pythorch, PyG, etc. So now I have
reinstalled everything, and I hope everything runs.

thanks in advance
Massimo

**Comment by rkansal47** [comment]
Hi, did reinstalling everything solve this issue?

**Comment by MaXDL4Phys** [comment]
Hi Raghav,

I'm Massimo, a PhD student at the University of Trento in collaboration
with Maurizio. I have started to work on your paper about MPGAN, and I'm to
reproduce your results.
Now It's running, my first training.
I reinstalled it, but I changed it a bit the Dockerfile. (I attach my
Dockerfile)
I had a bug on a line of plotting.py

> plt.tight_layout(2.0)

 and I changed it at line 89
 plt.tight_layout()

And now it's running.

thanks in advance
mb
